### ISSUE TITLE
Rules & Regulations FAQ

### ISSUE THREAD

**Issue by PerseusInLSC** [PerseusInLSC/Perseus-Study-Blog#8]
## What are the rules & requlations of Perseus' Study?
Rules & Regulations: https://sites.google.com/lsc.hk/perseus-study/about-us/copy-of-rules-regulations
## Why there is Rule 1: Confidentiality of Study Materials?
Our group's aim is to share information as much as we can, but in the process, we could imagine some classmates will take these resources, and sell it to others, and when this act is discovered by teachers, they might blame us (Perseus' Study).
To solve that problem, please, if anyone ask you or any members for resources, direct them to me.
Or at least don't try to make profit out of that, this rule (Confidentiality of Study Materials) is only semi-enforced.
P.S. If you're wondering why don't we just open all resources to the entirery of LSC, think carefully for a second (Copyright infringement issues).
> [!NOTE]
> You could copy all our resources and set up a new group (as we're using CC0), but it will be nice if you would notify us.
## What does type 2 rules mean?
In short, behavio(u)r issues.
We will add more to this article in the future.
